clients/flash/air-client/src/org/bigbluebutton/air/common/views/NoTabView.as

```diff
@@ -20,12 +22,18 @@ package org.bigbluebutton.air.common.views {
 			_topToolbar = createToolbar();
 			_topToolbar.percentWidth = 100;
 			addElement(_topToolbar);
+			addEventListener(Event.ADDED_TO_STAGE, onAddedToStage, false, 0, true);
+		}
+		
+		protected function onAddedToStage(event:Event):void {
+			stage.addEventListener(StageOrientationEvent.ORIENTATION_CHANGE, onOrientationChange, false, 0, true);
```

Comment:
I'm not a fan of adding these temporary listeners in every View. Is there an advantage to doing in this way instead of having the listener added at the Application level and having it exist for the life of the application?